### DISCUSSION TITLE
Castle

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1249] [Maps and Content Showcase]
Suitable for all modes except race.

**Comment by ghost** [comment]
[Castle.zip](https://github.com/redeclipse/base/files/6182130/Castle.zip)

**Comment by ghost** [comment]
![Screenshot 2021-03-22 20-15-47](https://user-images.githubusercontent.com/77372934/111990378-318d8400-8b4e-11eb-9881-54bbc7111400.png)
![Screenshot 2021-03-22 20-15-57](https://user-images.githubusercontent.com/77372934/111990395-33efde00-8b4e-11eb-9745-c3ba2118e367.png)
![Screenshot 2021-03-22 20-16-02](https://user-images.githubusercontent.com/77372934/111990405-35b9a180-8b4e-11eb-92e5-cc1864e14167.png)
![Screenshot 2021-03-22 20-16-07](https://user-images.githubusercontent.com/77372934/111990410-36523800-8b4e-11eb-9fa6-f1a1957224d4.png)
![Screenshot 2021-03-22 20-16-12](https://user-images.githubusercontent.com/77372934/111990416-37836500-8b4e-11eb-8cbd-9deba804a989.png)
![Screenshot 2021-03-22 20-16-22](https://user-images.githubusercontent.com/77372934/111990429-381bfb80-8b4e-11eb-9409-75ff085ef153.png)
![Screenshot 2021-03-22 20-16-27](https://user-images.githubusercontent.com/77372934/111990431-394d2880-8b4e-11eb-8c91-657cecb96b72.png)
![Screenshot 2021-03-22 20-16-38](https://user-images.githubusercontent.com/77372934/111990437-3a7e5580-8b4e-11eb-9a35-93a6670bf673.png)

**Comment by cloudziq** [comment]
nice you fixed it :)  no more lighting errors. Also you put way more details into it, which is a good thing too :)
Hope to see more maps from you and never stop practicing ^.^

**Reply by ghost** [reply]
Thank you.

**Comment by ghost** [comment]
[Castle.zip](https://github.com/redeclipse/base/files/6197897/Castle.zip)
This is the fixed version, some wrong indicative lines have been corrected.
